resources_pages/check-setup-mds.sh

```diff
@@ -30,16 +32,16 @@ if [[ "$(uname)" == 'Linux' ]]; then
     grep "Architecture" <<< $sys_info | sed 's/^[[:blank:]]*//;s/:/:    /' >> check-setup-mds.log
     grep "Kernel" <<< $sys_info | sed 's/^[[:blank:]]*//;s/:/:          /' >> check-setup-mds.log
     file_browser="xdg-open"
-    if ! $(grep -iq "20.04" <<< $os_version); then
+    if ! $(grep -iq "22.04" <<< $os_version); then
         echo '' >> check-setup-mds.log
-        echo "MISSING You need Ubuntu 20.04." >> check-setup-mds.log
+        echo "MISSING You are recommended to use Ubuntu 22.04." >> check-setup-mds.log
     fi
 elif [[ "$(uname)" == 'Darwin' ]]; then
     sw_vers >> check-setup-mds.log
     file_browser="open"
-    if ! $(sw_vers | grep -iq "11.[4|5]"); then
+    if ! $(sw_vers | grep -iq "12.\|11.[4|5|6]"); then
```

Comment:
@flor14 Could you confirm that this works on your Mac and it is no longer marked as "MISSING" on the top of the script output?

Reply:
For Mac is everything working except this 2 lines:

```
MISSING   rmarkdown PDF-generation failed. Check that latex and rmarkdown are marked OK above.
MISSING   rmarkdown HTML-generation failed. Check that rmarkdown is marked OK above.
```
and it is weird because rmarkdown it is working fine.

To be more complete could include Quarto, but it is already really good as it is anyway! Thanks Joel!

Reply:
Great that most things are working! To fix the PDF/HTML export, could you run these lines and tell me what the output is?

```
touch mds-knit-pdf-test.Rmd
Rscript -e "rmarkdown::render('mds-knit-pdf-test.Rmd', output_format = 'pdf_document')"
```

And then also run this one to see if it is the same error for html and pdf

```
Rscript -e "rmarkdown::render('mds-knit-pdf-test.Rmd', output_format = 'html_document')"
```

Reply:
For some reason pandoc is not working from the terminal


<img width="571" alt="image" src="https://user-images.githubusercontent.com/29610858/184729969-c155cf52-9cf9-473b-8115-a8b0604fb966.png">

Reply:
Ok, I fixed it installing Pandoc. [Here I found](https://community.rstudio.com/t/rmarkdown-pandoc-error-after-update-to-macos-catalina-10-15/43507/3) a potential explanation of what can be going wrong. Now the 3 installations are working!

```
# MDS setup check 2022.1

If a program or package is marked as MISSING,
this means that you are missing the required version of that program or package.
Either it is not installed at all or the wrong version is installed.
The required version is indicated with a number and an asterisk (*),
e.g. 4.* means that all versions starting with 4 are accepted (4.0.1, 4.2.5, etc).

You can run the following commands to find out which version
of a program or package is installed (if any):
```
name_of_program --version  # For system programs
conda list  # For Python packages
R -q -e "installed.packages()[,c(Package, Version)]"  # For R packages
```

Checking program and package versions...

## Operating system
ProductName:	macOS
ProductVersion:	12.5
BuildVersion:	21G72

## System programs
OK        psql (PostgreSQL) 14.5
OK        rstudio 2022.07.1+554
OK        R 4.2.1 (2022-06-23) -- "Funny-Looking Kid"
OK        python 3.10.5
OK        conda 4.13.0
OK        bash 3.2.57(1)-release (arm64-apple-darwin21)
OK        git 2.32.1 (Apple Git-133)
OK        make 3.81
OK        latex 3.141592653-2.6-1.40.24 (TeX Live 2022)
OK        tlmgr 58:07 +0200)
OK        docker 20.10.17, build 100c701
OK        code 1.70.1

## Python packages
OK        pandas=1.4.3
OK        pyppeteer=1.0.2
OK        nbconvert=6.4.4
OK        jupyterlab=3.4.5
OK        jupyterlab-git=0.38.0
OK        jupytext=1.14.0
OK        jupyterlab-spellchecker=0.7.2
OK        jupyterlab PDF-generation was successful.
OK        jupyterlab WebPDF-generation was successful.
OK        jupyterlab HTML-generation was successful.

## R packages
OK        tidyverse=1.3.2
OK        markdown=1.1
OK        rmarkdown=2.14
OK        renv=0.15.5
OK        IRkernel=1.3
OK        tinytex=0.40
OK        janitor=2.1.0
OK        gapminder=0.3.0
OK        readxl=1.4.0
OK        ottr=1.1.3
OK        canlang=0.0.1
OK        rmarkdown PDF-generation was successful.
OK        rmarkdown HTML-generation was successful.

The above output has been saved to the file /Users/florenciadandrea/Documents/check-setup-mds.log
together with system configuration details and any detailed error messages about PDF and HTML generation.
You can open this folder in your file browser by typing `open .` (without the surrounding backticks).

```

Reply:
Let's continue the M1 pandoc discussion in https://github.com/UBC-MDS/UBC-MDS.github.io/pull/139/files, I added some comments there.